rgw/v2/tests/s3_swift/test_sts_using_boto.py

```diff
@@ -74,24 +78,11 @@ def test_exec(config):
 
     auth = Auth(user1)
     iam_client = auth.do_auth_iam_client()
-    """
-    TODO:
-    policy_document and role_policy can be used valid dict types.
-    need to explore on this. 
-    """
-
-    policy_document = (
-        '{"Version":"2012-10-17",'
-        '"Statement":[{"Effect":"Allow","Principal":{"AWS":["arn:aws:iam:::user/%s"]},'
-        '"Action":["sts:AssumeRole"]}]}' % (user2["user_id"])
-    )
 
-    role_policy = (
-        '{"Version":"2012-10-17",'
-        '"Statement":{"Effect":"Allow",'
-        '"Action":"s3:*",'
-        '"Resource":"arn:aws:s3:::*"}}'
-    )
+    policy_document = json.dumps(config.sts["policy_document"]).replace(" ", "")
```

Comment:
In continuation to the above suggestion, we could use `yaml.safe_load(config.sts["policy_document"])`

Reply:
replied above